src/test/java/domain/RankTest.java

```diff
@@ -26,7 +24,29 @@ class RankTest {
 
         assertThat(Rank.fromResult(2, true)).isEqualTo(Rank.NONE);
         assertThat(Rank.fromResult(2, false)).isEqualTo(Rank.NONE);
+    }
+
+    @Test
+    void NONE인_경우_false를_반환한다() {
+        Rank rank = Rank.NONE;
+        assertThat(rank.isNotNone()).isEqualTo(false);
+    }
+
+    @Test
+    void NONE이_아닌_경우_false를_반환한다() {
+        Rank rank = Rank.FIFTH;
+        assertThat(rank.isNotNone()).isEqualTo(true);
```

Comment:
```suggestion
        assertThat(rank.isNotNone()).isTrue();
```
isEqualTo 외에도 다양한 메서드가 있어서 활용해보면 좋아요